lanebreach-api/db/migrate/20181129050151_create_sf311_case_metadata_table.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSf311CaseMetadataTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sf311_case_metadata do |t|      
```

Comment:
What are your thoughts on simply adding `bike_lane_type` and other metadata fields to the `sf_311_case` schema instead of creating an additional table?

Reply:
No strong preference either way. My thought process was to keep it separate, treating the `sf311_case` data as basically immutable + replica of the import source, and then add our data elsewhere.

This table's still a work-in-progress (like just finding out using `barrier` or `facility_t` is better than `bike_lane_type), and actually just added a reference to the `bikeway_network` table too. So will keep it separate for now as I think it'll be easier to iterate, but really not a strong preference.

Reply:
Got it, sounds good.